include/cutlass/epilogue/collective/builders/xe_builder.inl

```diff
@@ -118,10 +118,20 @@ namespace detail {
         CopyOpG2R
       >;
   };
-}
 
-  // Intel epilogue builder
+template <typename T>
+struct is_rowmajorish {
+  static_assert(cute::is_tuple_v<T>, "expected a tuple");
```

Comment:
What happens if you do `is_rowmajorish<cutlass::layout::ColumnMajor>`? It seems like it'd fail this static assert?